compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -2016,6 +2031,90 @@ fn add_order_independent_options(
     add_rpath_args(cmd, sess, codegen_results, out_filename);
 }
 
+// Write the debugger visualizer files for each crate to the target directory and gather the file paths.
+fn collect_debugger_visualizers(
+    out_filename: &Path,
+    sess: &Session,
+    codegen_results: &CodegenResults,
+    debugger_visualizer_paths: &mut Vec<PathBuf>,
+) {
+    let visualizers_out_dir = &out_filename.parent().unwrap().join("visualizers");
```

Comment:
Cargo won't be able to clean up this directory and won't copy it from the deps dir to the target dir. Are the debugger visualizers embedded in the executable itself? If so can you use a temporary directory to write them to instead?

Reply:
Using a temp directory to store the visualizer files instead of the deps dir.